.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+matrix:
+  include:
+    - node_js: "4.4"
+    - node_js: "5"
+    - node_js: "6"
+    - node_js: "7"
+sudo: required
+dist: trusty
+cache:
```

Comment:
What do you think about `cache: yarn` ?
https://docs.travis-ci.com/user/caching/#yarn-cache